src/main/java/com/google/sps/data/PersonRequest.java

```diff
@@ -64,21 +66,14 @@ public String getLinkedIn() {
     return qualifiedJobs;
   }
 
+  public boolean getOkShadow() {
+    return okShadow;
+  }
+
   public String toString() {
     return String.format(
-            "%s= %s:%s, %s:%s, %s:%s, %s:%s, %s:%s, %s: \n",
-            "PutPersonRequest",
-            "firstName",
-            firstName,
-            "lastName",
-            lastName,
-            "company",
-            company,
-            "job",
-            job,
-            "linkedIn",
-            linkedIn,
-            "qualifiedJobs")
+            "PutPersonRequest= firstName:%s, lastName:%s, company:%s, job:%s, linkedIn:%s, okShadow: %b, qualifiedJobs: \n",
+            firstName, lastName, company, job, linkedIn, okShadow)
         + qualifiedJobsToString();
```

Comment:
This is weird, is there a reason qualifiedJobsToString() can't be within String.format?

Reply:
Also, jobs should be before shadow to be consistent throughout.

Reply:
Just realized qualifiedJobs is printed nicely, you were right.